sips/sip-02X/sip-02X-wallet-interface.md

```diff
@@ -218,7 +221,232 @@ The following definitions can be used in multiple methods (mainly for transfer a
 
 - `profile`: `object` updated Schema.org Person object
 
-## Provider Registration
+## JS Representations
```

Comment:
Will definitely provide a better dev experience by making the values easier to inspect and transmit. Looking forward to using these JS representations.